test/unit/deferred.js

```diff
@@ -884,7 +884,9 @@ QUnit.test( "jQuery.when(thenable) - like Promise.resolve", function( assert ) {
 
 	var customToStringThen = {
 		then: function( onFulfilled ) {
-			onFulfilled();
+			// Support: Android 4.0 only
+			// Strict mode functions invoked without .call/.apply get global-object context
+			onFulfilled.call();
```

Comment:
Is there a story behind this? Just wondering how you found it out.

Reply:
This comment is copied from the `defaultContext` definition added by @gibson042 below. That `.call()` was just missing in this call (pun definitely intended).

This means strict mode in Android 4.0 is broken but we can't really do anything about it so this is just a workaround for the tests.

Reply:
Ah OK, now I get it.